eWonConnector-gateway/src/main/java/com/hms_networks/SyncManager.java

```diff
@@ -901,6 +907,29 @@ public QualityCode write(TagPath tagPath, Object o) {
                             provider.updateValue(p, v.getValue(), v.getQuality(), v.getTimestamp());
                         }
 
+                        // Sync tag description to documentation and tooltip
+                        try {
+                            // Get tag provider from gateway context
+                            TagProvider gwTagProvider = gatewayContext.getTagManager().getTagProvider(providerName);
+
+                            // Get path to tag and read config
+                            TagPath tagPath = com.inductiveautomation.ignition.common.tags.paths.parser.TagPathParser.parse(p);
+                            List<TagConfigurationModel> tagConfig = gwTagProvider.getTagConfigsAsync(Collections.singletonList(tagPath), false, true).get(5, TimeUnit.SECONDS);
+                            TagConfigurationModel tagConfigModel = tagConfig.get(0);
```

Comment:
Is zero being used to grab the first element or is the first element a special unique value?

Reply:
It's being used to grab the first element, since the getTagConfigsAsync method is taking in a singleton list

Reply:
It might be beneficial to give 0 a name that describes whats in that index. Would help with the readability.

Reply:
added